data/help.txt

```diff
@@ -94,6 +94,19 @@ help "multiple ship controls"
 	`Escorts can also be ordered with the mouse. Right clicking on an empty point in space will order escorts to fly to and hold position at that point, while right clicking on a hostile ship will order your escorts to focus fire on it, and right clicking on a friendly target will cause them to follow it.`
 	`Escorts can be saved into control groups by pressing Control+number key. All selected escorts will then be bound to that number key, selecting the group when pressed.`
 
+help "fleet asteroid mining"
+	`With an asteroid scanner and a fleet, you can direct your fleet to assist with mining operations and collect mined resources.`
+	`If your escort or fighter has no weapons but has cargo space, then it will chase after mined resources instead of mining.`
+
+help "fleet asteroid mining shortcuts"
+	`Shortcuts for asteroid targeting:`
+	`<Select nearest hostile ship> will select the nearest asteroid when no hostiles are nearby.`
```

Comment:
I'm not reusing R.  This is how the game works today (R targets asteroids and has as long as I have played).  I've made no changes to how the player actually does targeting.

The change I made is once the asteroid is targeted that F works the same way as it would attacking the ship.  I think changing the targeting is out of scope for this change.  But I can open a separate pull request for this if changing around shortcuts that already exist is desirable.

Reply:
I've decided I will implement this shortcut after all.  I'm going to make the shortcut V.

By pressing V you will select the nearest asteroid and your fleet will enter HARVEST mode.

Reply:
I have pushed commit e73d4ca8862c30922c89349337339f984b5217a4 which uses the V key instead of the R key.

Since some players are used to R for targeting asteroids at this point R will continue to target asteroids when there's no hostiles in the system.